web/src/app/shared/components/private-public-project/private-public-project.component.ts

```diff
@@ -40,7 +40,8 @@ export class PrivatePublicProjectComponent implements OnInit, OnDestroy {
     if (this.router.url === '/') {
       this.projectSubscription = this.projectService
         .findPublicProjects()
-        .subscribe((projects: IProject[]) => this.projects = projects.map((project: IProject) => new ProjectModel(project)));
+        .subscribe((projects: IProject[]) => this.projects = projects.map((project: IProject) => new ProjectModel(project))
+          .filter((project: ProjectModel) => project.repositories.length > 0));
```

Comment:
When the list get longer in the future it would be more efficient to get the database to do the filtering

Reply:
It does not look like it is possible to do a `repositories.length > 0` query in Firestore. We would need to store the `repositories` count to do this.

Best to store the repository length, so we can do a query to Firestore.

Reply:
Okay Do I have to make this change in this issue ? or we can do it in another ?

Reply:
I think do it in this PR, as this PR is tiny

Reply:
okay I will do it here